src/components/OpenSeadragonTileSource.js

```diff
@@ -11,7 +11,7 @@ export default function OpenSeadragonTileSource({
   const tiledImage = useRef(undefined);
 
   useEffect(() => {
-    if (!opacity) return;
+    if (opacity === undefined) return;
```

Comment:
I just got burned by strict equality to undefined somewhere else:

```suggestion
    if (opacity == null) return;
```